.github/actions/runleaks/README.md

```diff
@@ -0,0 +1,163 @@
+# runleaks
+
+[![Scan Action Logs](https://github.com/JosiahSiegel/runleaks/actions/workflows/main.yml/badge.svg?branch=main)](https://github.com/JosiahSiegel/runleaks/actions/workflows/main.yml)
```

Comment:
needs update

Reply:
All these issues are not documented and we are in the process of reaching out to Mr. Josiah Siegel and ask for his assistance in figuring out the different environment and secrets we need to setup in this project.